Unity-Technologies/com.unity.multiplayer.samples.coop
# In editor tutorials: Welcome dialog and table of contents [MTT-8698], [MTT-8694]

### Description
- Added a welcome dialog to guide users on their first time experience
- Added a Table of Contents with helpful resources, that can be accessed from the `Tutorials > Show Tutorials` menu

![image](https://github.com/user-attachments/assets/46d7e4f6-1b1f-476d-a3cc-03a8537cad1a)

![image](https://github.com/user-attachments/assets/1a4ccc40-fa1e-4b47-8f86-47bd11711f46)


### Issue Number(s)
- https://jira.unity3d.com/browse/MTT-8694
- https://jira.unity3d.com/browse/MTT-8698

## How to test/validate
1. Clone this branch
2. Open it with Unity
3. The welcome dialog will show up
4. Go to Tutorials > Show Tutorials to open the Table of Contents
5. Click all the links to ensure they work

### Contribution checklist
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
